war/src/main/webapp/help/system-config/master-slave/availability_fr.html

```diff
@@ -9,25 +9,27 @@
             C'est la configuration par défaut et la plus classique.
             Dans ce mode, Jenkins fait des tests réguliers pour conserver l'agent
             actif.
+            <p>
             Si Jenkins peut démarrer l'agent sans l'aide de l'utilisateur et
             que l'agent n'est pas disponible,
             il tentera régulièrement de le relancer.
             Jenkins ne cherchera pas à le désactiver.
         </dd>
 
-        <!--dt><b>
-            Arrêter et démarrer l'agent à des moment spécifiques
+        <dt><b>
+            Activer l'agent à des moments spécifiques
         </b></dt>
         <dd>
             Dans ce mode, Jenkins conservera l'agent activé selon les horaires configurés.
-            Si Jenkins peut démarrer l'agent sans l'aide de l'utilisateur et
-            que l'agent n'est pas disponible dans la fenêtre où cet
-            il est censé être activé,
-            il tentera régulièrement de le relancer.
-
-            Pendant la fenêtre où l'agent est censé être désactivé, il ne le sera
-            réellement que s'il n'a pas de job en cours.
-        </dd-->
+            <p>
+            Si l'agent n'est pas disponible dans la fenêtre où il est censé être activé,
```

Comment:
J'ai du mal avec la fenêtre là, c'était la fenêtre de temps peut être ??

```suggestion
            Si l'agent n'est pas disponible dans au moment où il est censé être activé,
```